.github/workflows/_build_artifacts.yml

```diff
@@ -239,16 +227,6 @@ jobs:
           service_account: "github-actions@github-iam-387915.iam.gserviceaccount.com"
           export_environment_variables: true
           create_credentials_file: true
-      - name: Copy binaries to Google Cloud Storage
```

Comment:
We need to keep this one for our prod Gateway installation https://github.com/firezone/firezone/blob/main/terraform/environments/production/gateways.tf#L28 to work and be able to install version for a specific github sha.

Reply:
Fixed